README.md

```diff
@@ -375,6 +375,8 @@ the following events:
 * `"tutorialVisibilityChange"` - triggered when the tutorial preview is turned on or off. It includes an `Object` with a `visibility` property that indicates whether the tutorial is visible.
 * `"inspectorChange"` - triggered whenever the inspector changes from enabled to disabled, or vice versa. It includes an `Object` with an `enabled` property set to `true` or `false`.
 * `"autoUpdateChange"` - triggered whenever the auto update preference changes from enabled to disabled, or vice versa. It includes an `Object` with a `autoUpdate` property set to `true` or `false`
+* `"projectDirty"` - triggered whenever the current file has been changed. It includes an `Object` with the path to the current file.
```

Comment:
This isn't quite accurate.  This will be triggered when one of the files in the project has been edited and those changes haven't been saved yet.  Also, change "path" to "`path`" (add backticks around it).